docs/_layouts/page.html

```diff
@@ -63,6 +62,21 @@ <h1 class="post-title-main">{{ page.title }}</h1>
         {% endif %}
     </div>
 
+    {% if site.github_editme_path %}
+        This documentation is written in markdown. If there is something missing or can be improved, edit this page on github and create a PR: 
+        {% unless page.editmepath == false %}
+        <a target="_blank" href="https://github.com/{{site.github_editme_path}}{{editmepath}}" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>
+        <script>
+            $(document).ready ( function(){
+                $('div.post-content').find('h2').after(
+                '  <a target="_blank" href="https://github.com/{{site.github_editme_path}}{{editmepath}}" class="btn btn-default btn-xs githubEditButton" role="button"><i class="fa fa-github fa-lg"></i></a>'
```

Comment:
Instead of this (a button):
![grafik](https://user-images.githubusercontent.com/1573684/156597539-c77f30ab-760d-41f7-b95b-1655746a86e6.png)

I'd like to have something like this:
![grafik](https://user-images.githubusercontent.com/1573684/156597457-845da9e3-71e1-46e9-8ed5-a6750defa897.png)

I'm not sure, whether we should limit it to "h2" only. It seems, that we currently use h2 for our headings after the title, which we render as h1. But maybe we should include at least "h3"?

Naturally this js snippet belongs somewhere under js, e.g. into js/customscripts.js or in a new file. The problem is, that we don't have access to the page in the js file. That means, we need to exchange data. One simple way could be, to add this information to the surrounding div, like this: `<div class="post-content" data-github-edit-url="https://github.com/{{site.github_editme_path}}{{editmepath}}">`. Then we could get this information from the node in our javascript code.

Reply:
I think, that's fine (after the anchors).